src/settings/settings.vcxproj

```diff
@@ -60,7 +60,7 @@
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
       <AdditionalIncludeDirectories>..;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <AdditionalOptions>/Zm150 %(AdditionalOptions)</AdditionalOptions>
+      <AdditionalOptions>/Zm250 %(AdditionalOptions)</AdditionalOptions>
```

Comment:
couldn't compile again w/o increasing the memory limit :< we might probably clean up the PCH later.